bootstrap.py

```diff
@@ -138,19 +142,20 @@ def __init__(self, cache_dir: Path) -> None:
 
     @staticmethod
     def get_file_hash(path: Path) -> str:
-        with open(path, "rb") as file:
-            bytes = file.read()
-            readable_hash = hashlib.sha256(bytes).hexdigest()
-            return readable_hash
+        """Get the hash of a file.
+        Returns an empty string if the file does not exist."""
+        if path.is_file():
+            with open(path, "rb") as file:
+                bytes = file.read()
+                readable_hash = hashlib.sha256(bytes).hexdigest()
+                return readable_hash
+        else:
+            return ""
```

Comment:
This was added to be able to handle files which are no longer required as dependencies or are newly added.